src/shared/dissect-image.h

```diff
@@ -63,6 +63,7 @@ typedef enum DissectImageFlags {
         DISSECT_IMAGE_NO_UDEV             = 1 << 9,  /* Don't wait for udev initializing things */
         DISSECT_IMAGE_RELAX_VAR_CHECK     = 1 << 10, /* Don't insist that the UUID of /var is hashed from /etc/machine-id */
         DISSECT_IMAGE_FSCK                = 1 << 11, /* File system check the partition before mounting (no effect when combined with DISSECT_IMAGE_READ_ONLY) */
+        DISSECT_IMAGE_NO_GPT              = 1 << 12, /* Only recognize single file system images */
```

Comment:
i think we should name this more precisely, people might think this would just turn off gpt and leave mbr partition table support on. i.e. maybe call this DISSECT_IMAGE_NO_PARTITION_TABLE? or DISSECT_WITHOUT_PARTITION_TABLE or so?

Reply:
Makes sense, done

Reply:
Makes sense, done